apps/coscrad-frontend/src/store/slices/resources/shared/notes-for-resource/self-note.presenter.tsx

```diff
@@ -1,15 +1,28 @@
+import styled from '@emotion/styled';
+import { TextSnippet as TextSnippetIcon } from '@mui/icons-material';
+import { Box, Grid, Typography } from '@mui/material';
 import { SelfConnectionNote } from '../../../notes/hooks/use-loadable-self-notes-for-resource';
 
+const Item = styled(Box)(({ theme }) => ({
+    padding: 0,
+    marginBottom: '15px',
+}));
+
 export const SelfNotePresenter = ({ text, id, context }: SelfConnectionNote): JSX.Element => (
-    <div data-testid={id}>
-        {/* TODO Use property presenter helper after rebasing */}
-        <strong>Note ({id}):</strong>
-        <p>{text}</p>
-        <br />
-        {/* TODO Eventually we should offer a way to highlight this context in the detail view */}
-        {/* We may not need to present this here in this way. This is for development only. */}
-        <strong>Context</strong>
-        {JSON.stringify(context)}
-        <br />
-    </div>
+    <Item data-testid={id}>
```

Comment:
`Item` could be a `StackItem` component that could serve as the item for any of the Stacks in these drawers.